### ISSUE TITLE
flutter doctor exited unexpectedly

### ISSUE THREAD

**Issue by srisusil** [flutter/flutter#16108]
## Steps to Reproduce

Please tell us what you were doing and what went wrong. If you are running flutter tools from the command line, please try adding the `-v` or `--verbose` option to gather more information.

If the problem is with your application's rendering, please attach a screenshot and any relevant source code.
If you are getting an exception in the logs, and your code is implicated in the first few frames, then please include the source code for the functions involved.

## Logs

Run your application with `flutter run` and attach all the log output.

Run `flutter analyze` and attach any output of that command also.

## Flutter Doctor

Paste the output of running `flutter doctor -v` here.

> For more information about diagnosing and reporting Flutter bugs, please see [https://flutter.io/bug-reports/](https://flutter.io/bug-reports/).

**Comment by estebancores** [comment]
[flutter_01.log](https://github.com/flutter/flutter/files/1864672/flutter_01.log)

Im having a problem with my flutter, i dont know how to fix it!. help me please.

**Comment by xster** [comment]
From your log file, it looks like line 1 of the .git/config file is unparsable by git. Can you copy paste the contents of your .git/config?

**Comment by estebancores** [comment]
if i open that file, only has a line with "null" repeated like 30 times, what can i do then?

**Comment by xster** [comment]
It's pretty weird that your .git/config has null printed in it. Do you know how it happened or whether it was manually edited?

To restore your repo, you may have to do:
```bash
git remote add upstream git@github.com:flutter/flutter.git
git fetch
git co beta
```

Also, just for our info, when did you originally clone the repo?

**Comment by estebancores** [comment]
i really don't know  why it has null pinted in it. I get flutter almost like February but i don't clone it, I just downloaded it. already it's working again, i apreciate your help, thank you very much.

**Comment by jwendel** [comment]
Make sure your Android SDKs are up-to-date.
https://developer.android.com/studio/intro/update.html
https://developer.android.com/studio/command-line/sdkmanager.html

**Comment by srisusil** [comment]
flutter, Android, dart everything is up to date. But , when I run "flutter doctor" It simply shows "flutter exited unexpectedly".
[flutter_10.log](https://github.com/flutter/flutter/files/1866070/flutter_10.log)


Terminal Output:
$flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.2.3, on Linux, locale C.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
[✓] Android Studio (version 3.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2018.1)
[✓] Connected devices (1 available)

• No issues found!

Oops; flutter has exited unexpectedly.
Sending crash report to Google.
Failed to send crash report due to a network error: SocketException: Failed host lookup: 'clients2.google.com' (OS Error: Temporary failure in name resolution, errno = -3)
Crash report written to /home/dracula/flutter_10.log;
please let us know at https://github.com/flutter/flutter/issues.

**Comment by xster** [comment]
@cbracken, vaguely remember you talking about non-standard UTF terminals

**Comment by tvolkert** [comment]
A locale of `C.UTF_8`??

Something in your environment setup seems very weird.  Can you please run `flutter doctor --bug-report` and attach the resulting ZIP file?

**Comment by BondarenkoStas** [comment]
@srisusil was the issue fixed?

**Comment by BondarenkoStas** [comment]
I believe the issue was fixed in 0.5.0 with https://github.com/flutter/flutter/pull/17877. Please correct me if I'm wrong.
